experimental/dds/tree2/src/test/feature-libraries/contextuallyTyped.spec.ts

```diff
@@ -165,9 +165,11 @@ describe("ContextuallyTyped", () => {
 				libraries: [leaf.library],
 			});
 
-			const nodeSchema = builder.objectRecursive("node", {
+			const recursiveReference = () => nodeSchema;
+			builder.fixRecursiveReference(recursiveReference);
```

Comment:
Oops, I forgot to revert this change since this pattern tends to crash the stop working if you use the resulting schema very much. I think it ends up being order dependent across files in some broken way or something.

Reply:
I guess this one works so I can leave it.